modules/admin/con-rhdh-operator-deployment-config.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assembly:
+// assembly-rhdh-operator-deployment-config
+
+[id="con-rhdh-operator-deployment-config_{context}"]
+= {product} Operator deployment configuration
+
+Since v1alpha2 (Operator version 0.3.0), the Backstage Custom Resource Definition (CRD) contains `spec.deployment`. This allows for patching the Backstage Deployment resource with fields defined in `spec.deployment.patch`, which contains a fragment of the `apps.Deployment` object.
```

Comment:
```suggestion
As of 1.3, the Red Hat Developer Hub operator exposes a new `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the RHDH Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
```

Suggestion to reword the sentence.
Also, there is no such 0.3.0 version of the downstream operator. The downstream operator follows the version of RHDH, which is 1.3.